src/tye/InitHost.cs

```diff
@@ -6,21 +6,32 @@ namespace Microsoft.Tye
 {
     public static class InitHost
     {
-        public static string CreateTyeFile(FileInfo? path, bool force)
+        public static string CreateTyeFile(FileInfo? path, bool force, bool json = false)
         {
-            var (content, outputFilePath) = CreateTyeFileContent(path, force);
+            var (content, outputFilePath) = CreateTyeFileContent(path, force, json);
 
-            File.WriteAllText(outputFilePath, content);
+            if (json)
+            {
+                var jsonContent = YamlSerializer.ConvertToJson(content);
+                File.WriteAllText(outputFilePath, jsonContent);
+            }
+            else
+            {
+                File.WriteAllText(outputFilePath, content);
+            }
 
             return outputFilePath;
         }
 
-        public static (string, string) CreateTyeFileContent(FileInfo? path, bool force)
+        public static (string, string) CreateTyeFileContent(FileInfo? path, bool force, bool json = false)
         {
             if (path is FileInfo && path.Exists && !force)
             {
                 ThrowIfTyeFilePresent(path, "tye.yml");
                 ThrowIfTyeFilePresent(path, "tye.yaml");
+
+                if (json)
+                    ThrowIfTyeFilePresent(path, "tye.json");
```

Comment:
This check should probably happen regardless of `json` being true/false. We don't want to create a `tye.yaml` if you already have `tye.json` without `force`.